WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -237,25 +234,16 @@ def getQueues( self, resourceDict ):
             si00 = float( self.queueDict[queueName]['ParametersDict']['SI00'] )
             queueCPUTime = 60. / 250. * maxCPUTime * si00
             self.queueDict[queueName]['ParametersDict']['CPUTime'] = int( queueCPUTime )
-          queueTags = self.queueDict[queueName]['ParametersDict'].get( 'Tag' )
-          if queueTags and isinstance( queueTags, basestring ):
-            queueTags = fromChar( queueTags )
-            self.queueDict[queueName]['ParametersDict']['Tag'] = queueTags
-          if ceTags:
-            if queueTags:
-              allTags = list( set( ceTags + queueTags ) )
-              self.queueDict[queueName]['ParametersDict']['Tag'] = allTags
-            else:
-              self.queueDict[queueName]['ParametersDict']['Tag'] = ceTags
 
           maxMemory = self.queueDict[queueName]['ParametersDict'].get( 'MaxRAM', None )
+
           if maxMemory:
             # MaxRAM value is supposed to be in MB
             maxMemoryList = range( 1, int( maxMemory )/1000 + 1 )
             memoryTags = [ '%dGB' % mem for mem in maxMemoryList ]
             if memoryTags:
               self.queueDict[queueName]['ParametersDict'].setdefault( 'Tag', [] )
-              self.queueDict[queueName]['ParametersDict']['Tag'] += memoryTags
+              self.queueDict[queueName]['ParametersDict']['Tag'] += ','.join( memoryTags )
```

Comment:
I have a question here: the moment the SiteDirector is selecting if there are pilots to send, do we really need that TAGs become matching parameters? Because at this moment I would live them out.